.evergreen/config.yml

```diff
@@ -304,6 +303,15 @@ functions:
             export AWS_SECRET_ACCESS_KEY="${cse_aws_secret_access_key}"
             export AWS_DEFAULT_REGION="us-east-1"
             ${PYTHON3_BINARY} -m venv ./venv
+
+            # Set the PYTHON environment variable to point to the active python3 binary. This is used by the
+            # set-temp-creds.sh script.
+            if [ "Windows_NT" = "$OS" ]; then
+              export PYTHON="$(pwd)/venv/Scripts/python"
+            else
+              export PYTHON="$(pwd)/venv/bin/python"
+            fi
+
```

Comment:
Ideally, we'd combine the virtual environment creation, python3 binary setting, and boto3 installation into a single call to [activate_venv.sh.](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/activate_venv.sh). But, that unfortunately sets `PYTHON_BINARY` and not `PYTHON`, so it wouldn't work perfectly...

Reply:
Also, `PYTHON3_BINARY` points to the system installation of python3, but we actually want to point to the virtualenv python installation here.

Reply:
Ah yep, that too.